### ISSUE TITLE
No Sonic 2 Nick Arcade Object Format

### ISSUE THREAD

**Issue by AlexField442** [sonicretro/SonLVL#100]
I'm making a disassembly for _Sonic 2 Nick Arcade_, and I noticed that SonLVL doesn't have proper support for the object format used in that build (i.e., certain objects being viewed use wrong IDs, flipped flag being swapped, etc.).

**Comment by Caverns4** [comment]
Yes. Setting the object format to Sonic 1 is correct. It seems the version preset for "version=S2NA" is incorrect.
